modules/node/node_module.e

```diff
@@ -176,6 +186,9 @@ feature -- Hooks
 			create lnk.make ("List of nodes", a_response.url ("/nodes", Void))
 			a_menu_system.primary_menu.extend (lnk)
 
+			create lnk.make ("Trash nodes", a_response.url ("/trash", Void))
```

Comment:
here we could use permission "view trash"
to display the link or not 
most of the time, anonymous user would not need such menu.

non critical for now